src/main/kotlin/no/nav/syfo/client/oppfolgingstilfelle/OppfolgingstilfelleClient.kt

```diff
@@ -0,0 +1,59 @@
+package no.nav.syfo.client.oppfolgingstilfelle
+
+import io.ktor.client.call.*
+import io.ktor.client.plugins.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.http.*
+import net.logstash.logback.argument.StructuredArguments
+import no.nav.syfo.client.azuread.AzureAdClient
+import no.nav.syfo.client.httpClientDefault
+import no.nav.syfo.domain.*
+import no.nav.syfo.util.*
+import org.slf4j.LoggerFactory
+import java.util.UUID
+
+class OppfolgingstilfelleClient(
+    private val azureAdClient: AzureAdClient,
+    private val isoppfolgingstilfelleClientId: String,
+    isoppfolgingstilfelleBaseUrl: String,
+) {
+    private val personOppfolgingstilfelleSystemUrl: String =
+        "$isoppfolgingstilfelleBaseUrl$ISOPPFOLGINGSTILFELLE_OPPFOLGINGSTILFELLE_SYSTEM_PERSON_PATH"
+
+    private val httpClient = httpClientDefault()
+
+    suspend fun oppfolgingstilfelle(
```

Comment:
Jeg ville kanskje tenkt at navnet burde vært noe mer a la `getOppfolgingstilfelle`/ `getOppfolgingstilfellePerson` ? For nå ser det mer ut som en variabel (en ting) enn en funksjon (en handling).

Reply:
+1